test/e2e/parallel/nstemplatetier_test.go

```diff
@@ -154,16 +155,15 @@ func TestResetDeactivatingStateWhenPromotingUser(t *testing.T) {
 	awaitilities := WaitForDeployments(t)
 	hostAwait := awaitilities.Host()
 	t.Run("test reset deactivating state when promoting user", func(t *testing.T) {
-		userSignup, _ := NewSignupRequest(awaitilities).
+		user := NewSignupRequest(awaitilities).
 			Username("promoteuser").
 			Email("promoteuser@redhat.com").
 			ManuallyApprove().
 			TargetCluster(awaitilities.Member1()).
 			EnsureMUR().
 			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
-			Execute(t).
-			Resources()
-
+			Execute(t)
+		userSignup := user.UserSignup
 		// Set the deactivating state on the UserSignup
 		updatedUserSignup, err := hostAwait.UpdateUserSignup(t, userSignup.Name,
```

Comment:
in this case, you could do
```suggestion
		// Set the deactivating state on the UserSignup
		updatedUserSignup, err := hostAwait.UpdateUserSignup(t, user.UserSignup.Name,
```